invenio_records_rest/views.py

```diff
@@ -103,10 +117,16 @@ def create_blueprint(endpoints):
         url_prefix='',
     )
 
+    error_handlers_registry = defaultdict(dict)
     for endpoint, options in (endpoints or {}).items():
+        error_handlers = options.pop('error_handlers', None)
```

Comment:
could this be ``options.pop('error_handlers', {})`` instead? If so, remove ``(error_handlers or {})`` below.